dask/config.py

```diff
@@ -68,10 +68,10 @@ def update(old, new, priority='new'):
     dask.config.merge
     """
     for k, v in new.items():
-        if k not in old and type(v) is dict:
+        if k not in old and isinstance(v, Mapping):
```

Comment:
Should we check for `MutableMapping`s?

Reply:
No need, I think.  The items in `new` are just copied (not modified), so `v` need not be mutable.